spec/features/foods/index_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'food search page' do
+   it 'see a total of the number of items returned by the search' do
+      visit root_path
+      fill_in :q, with: "sweet potatoes"
+      click_on "Search"
+
+      expect(current_path).to eq(foods_path)
+      expect(page).to have_content("Total number of items found: 10")
```

Comment:
Instead of listing the total items we're displaying (which will always be 10) we actually wanted you to display the total number of matches from the API, which is stored at the `totalHits` key in the json response. 

Figuring out how to capture that data is a bit tricky so I recommend thinking a bit about how you could grab that data point and make it accessible in the view